src/test/java/org/killbill/billing/plugin/adyen/AdyenPaymentPluginApiTest.java

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright 2020-2023 Equinix, Inc
+ * Copyright 2014-2023 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.adyen;
+
+import com.google.common.collect.ImmutableList;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Test;
+import org.killbill.billing.osgi.api.Healthcheck;
+import org.killbill.billing.payment.api.Payment;
+import org.killbill.billing.payment.api.PaymentApiException;
+import org.killbill.billing.payment.api.PaymentMethodPlugin;
+import org.killbill.billing.payment.api.PaymentTransaction;
+import org.killbill.billing.payment.api.TransactionType;
+import org.killbill.billing.payment.plugin.api.PaymentMethodInfoPlugin;
+import org.killbill.billing.payment.plugin.api.PaymentPluginApiException;
+import org.killbill.billing.payment.plugin.api.PaymentPluginStatus;
+import org.killbill.billing.payment.plugin.api.PaymentTransactionInfoPlugin;
+import org.killbill.billing.plugin.TestUtils;
+import org.killbill.billing.plugin.adyen.api.AdyenPaymentMethodInfoPlugin;
+import org.killbill.billing.plugin.adyen.api.AdyenPaymentMethodPlugin;
+import org.killbill.billing.plugin.adyen.core.AdyenHealthcheck;
+import org.killbill.billing.plugin.adyen.dao.gen.tables.records.AdyenPaymentMethodsRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AdyenPaymentPluginApiTest extends TestBase {
```

Comment:
AFAIK our convention for test name is `Test<ClassName>` instead of `<ClassName>Test`